src/utils.jl

```diff
@@ -118,47 +159,85 @@ Base.keys(q::UnionAbstractQuantity) = keys(ustrip(q))
 
 
 # Numeric checks
-function Base.isapprox(l::UnionAbstractQuantity, r::UnionAbstractQuantity; kws...)
-    l, r = promote_except_value(l, r)
-    return isapprox(ustrip(l), ustrip(r); kws...) && dimension(l) == dimension(r)
-end
-function Base.isapprox(l::Number, r::UnionAbstractQuantity; kws...)
-    iszero(dimension(r)) || throw(DimensionError(l, r))
-    return isapprox(l, ustrip(r); kws...)
-end
-function Base.isapprox(l::UnionAbstractQuantity, r::Number; kws...)
-    iszero(dimension(l)) || throw(DimensionError(l, r))
-    return isapprox(ustrip(l), r; kws...)
-end
-Base.iszero(d::AbstractDimensions) = all_dimensions(iszero, d)
-function Base.:(==)(l::UnionAbstractQuantity, r::UnionAbstractQuantity)
-    l, r = promote_except_value(l, r)
-    ustrip(l) == ustrip(r) && dimension(l) == dimension(r)
+for op in (:(<=), :(<), :(>=), :(>), :isless),
```

Comment:
For the record: I would have thought these operators would be automatically defined by `isless`, since that's what the docstring says. But it turns out that some packages will specifically check whether a method is defined on these (Turing.jl being an example).